citesphere/src/main/java/edu/asu/diging/citesphere/web/user/AsyncDeleteCitationsResponse.java

```diff
@@ -0,0 +1,36 @@
+package edu.asu.diging.citesphere.web.user;
```

Comment:
same as above, unless this class is used exclusively by controllers, it shouldn't be in the controller package.